### DISCUSSION TITLE
第4节课作业(7班)

### DISCUSSION THREAD

**Discussion by JimmyMa99** [InternLM/Tutorial#311] [作业]
第4节课作业(7班)

**Comment by wuhaostudio** [comment]
基础作业
<img width="1080" alt="微信图片_20240111194531" src="https://github.com/InternLM/tutorial/assets/143194075/bf687db7-f66e-4a32-91e5-89e2c9f6e409">

**Comment by createup2023** [comment]
https://www.yuque.com/creatup/ynrsde/yik4gypdy36ldt7c?singleDoc# 《第四次作业：》

![image](https://github.com/InternLM/tutorial/assets/128460366/a74b1d89-6804-4109-806f-f6dc349a3b10)
![image](https://github.com/InternLM/tutorial/assets/128460366/ae0a1544-a741-4469-a8ce-693444769993)

**Comment by leochiang** [comment]
<img width="669" alt="40240113190616" src="https://github.com/InternLM/tutorial/assets/7628117/cb1439ed-34e4-43af-af1a-49bb6a2e3eb3">

**Comment by hui1feng** [comment]
第四节课基础作业：https://github.com/hui1feng/XTuner-Finetune/blob/main/Xtuner-Finetune.ipynb

**Comment by SaaRaaS-1300** [comment]
第四节课 笔记同作业 github链接：https://github.com/SaaRaaS-1300/InternLM_openNotebook/blob/main/Lesson-4/Lesson-4-Notebook.md

**Comment by abzm11** [comment]
第四课笔记同作业：https://blog.csdn.net/weixin_38043453/article/details/135587465

 user_avator = "/root/personal_assistant/code/InternLM/doc/imgs/user.png"

robot_avator = "/root/personal_assistant/code/InternLM/doc/imgs/robot.png"

ssh -CNg -L 127.0.0.1:6006:0.0.0.0:6006 -o "StrictHostKeyChecking no" root@ssh.intern-ai.org.cn -p 33693
![image](https://github.com/InternLM/tutorial/assets/156595582/b367fc8a-57bf-4072-b5cd-1cf06d92172d)

**Comment by PommesPeter** [comment]
基础作业：
![image](https://github.com/InternLM/tutorial/assets/54879512/7f4d39e5-cf42-4cab-aeb7-a091f942d8fe)

**Comment by lifelinezhang** [comment]
https://zhuanlan.zhihu.com/p/677481624

**Comment by anneeeee** [comment]
![image](https://github.com/InternLM/tutorial/assets/16276686/348f77f1-7fae-4fe5-af4f-b49f6554df96)

**Comment by snowhou** [comment]
打卡：https://zhuanlan.zhihu.com/p/679042577

**Comment by Rogue-Trader-zzy** [comment]
打卡：https://www.yuque.com/roguetrader/iu9odz/quwoqk3edosumg9y?singleDoc# 《个人小助手认知微调实践》

**Comment by Simon-lyz** [comment]
## InternLM Lesson-4
https://g4w90egc30.feishu.cn/docx/K8qodGQ27ojqWrxJxIycyin6nRg?from=from_copylink
